test/1_utils.js

```diff
@@ -1,15 +1,19 @@
 'use strict';
-
+/*jshint -W079 */
+var sinon = require('sinon');
```

Comment:
I'm curious what else is defining `sinon`

Reply:
this is a rule conflict between google closure compiler and jshint rules.
In the near future, we are going to replace jshint with eslint and this re-definition issue won’t  be present.